packages/blade/src/components/Table/Table.web.tsx

```diff
@@ -121,9 +122,9 @@ const _Table = <Item,>({
   height,
   showStripedRows,
   gridTemplateColumns,
-  surfaceLevel = 2,
   isLoading = false,
   isRefreshing = false,
+  backgroundColor = 'surface.background.gray.intense',
```

Comment:
Should we also add a JS check to ensure no other colors are passed here?

Reply:
I don't think that would be necessary. 

The UI itself will look broken if they @ts-ignore and set any different color.